### PR TITLE
Spelling - Scatty: "Meine" (DE)

### PR DESCRIPTION
**Pull request by AmProsius**
**Describe the bug**
In the German localization there's a typo for a log entry concerning the defeat versus Kirgo.

**Changelog**
Im Tagebucheintrag nach dem Kampf gegen Kirgo die Phrase "Meinem Niederlage gegen Kirgo [...]" korrigiert zu "Meine Niederlage gegen Kirgo [...]".

### CODE REVIEW

**Comment by AmProsius** [comment]
The fix works (tested manually ingame), but the automatic test fails. Maybe you can have a look at it, @szapp?

**Comment by szapp** [comment]
> The fix works (tested manually ingame), but the automatic test fails. Maybe you can have a look at it, @szapp?

What’s the fail message?

**Comment by AmProsius** [comment]
> What’s the fail message?

`Log topic entry was not added by the dialog function`

File: src/Ninja/G1CP/Content/Tests/test133.d [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    // Check if it was updated
+    if (G1CP_LogHasEntry(topic, wrong)) {
+        G1CP_TestsuiteErrorDetail("Log topic entry (incorrect) remained unchanged");
+        passed = FALSE;
+    };
+    if (!G1CP_LogHasEntry(topic, right)) {
+        G1CP_TestsuiteErrorDetail("Log topic entry (correct) does not exist");
+        passed = FALSE;
+    };
+    G1CP_LogRemoveTopic(topic);
```
Comment (szapp) [inline src/Ninja/G1CP/Content/Tests/test133.d:82]: The log topic is removed here and is not created during the dialog info function (unlike in #93). What will be necessary is to create it anew (see the lines 65-68).

File: src/Ninja/G1CP/Content/patchInit.d [hunk 2]
```diff
@@ -63,6 +63,7 @@ func void Ninja_G1CP_Menu(var int menuPtr) {
         G1CP_122_CavalornDailyRoutine();                // #122
         G1CP_125_ButcherText();                         // #125
         G1CP_126_SharkyTrade();                         // #126
+        G1CP_133_DE_LogEntryScatty();                   // #133
```
Comment (szapp) [inline src/Ninja/G1CP/Content/patchInit.d:66]: This is a gamesave fix. It needs to be initialized from `G1CP_GamesaveFixes_Apply` and reverted in `G1CP_GamesaveFixes_Revert` not here.

**Review by szapp — CHANGES_REQUESTED** [review]

**Review by szapp — APPROVED** [review]
Seems to work fine. I didn't to elaborate testing (no saving and loading) as it should be the same as #93.

**Pull request merged**